src/test/scala/nl.knaw.dans.easy.solr4files/AppSpec.scala

```diff
@@ -69,24 +72,112 @@ class AppSpec extends TestSupportFixture {
     }
   }
 
+  private def anonymousAuthItem(uuid: UUID, path: String) = Success(
+    s"""{
+       |  "itemId":"$uuid/$path",
+       |  "owner":"someone",
+       |  "dateAvailable":"1992-07-30",
+       |  "accessibleTo":"ANONYMOUS",
+       |  "visibleTo":"ANONYMOUS"
+       |}""".stripMargin)
+
   "update" should "call the stubbed solrClient.request" in {
     initVault()
     assume(canConnectToEasySchemas)
-    val result = new StubbedSolrApp().update(storeName, uuidCentaur)
+    val app = new StubbedSolrApp()
+    app.expectsHttpAsString(anonymousAuthItem(uuidCentaur, "data/path/to/a/random/video/hubble.mpg")) once()
+    app.expectsHttpAsString(anonymousAuthItem(uuidCentaur, "data/reisverslag/centaur.mpg")) once()
+    app.expectsHttpAsString(anonymousAuthItem(uuidCentaur, "data/reisverslag/centaur.srt")) once()
+    app.expectsHttpAsString(anonymousAuthItem(uuidCentaur, "data/reisverslag/centaur-nederlands.srt")) once()
+    app.expectsHttpAsString(anonymousAuthItem(uuidCentaur, "data/reisverslag/deel01.docx")) once()
+    app.expectsHttpAsString(anonymousAuthItem(uuidCentaur, "data/reisverslag/deel01.txt")) once()
+    app.expectsHttpAsString(anonymousAuthItem(uuidCentaur, "data/reisverslag/deel02.txt")) once()
+    app.expectsHttpAsString(anonymousAuthItem(uuidCentaur, "data/reisverslag/deel03.txt")) once()
+    app.expectsHttpAsString(anonymousAuthItem(uuidCentaur, "data/ruimtereis01_verklaring.txt")) once()
+
+    val result = app.update(storeName, uuidCentaur)
```

Comment:
inline in next line?
if yes, same below

Reply:
If the test fails you have to extract again to figure out what you did get.